### DISCUSSION TITLE
Ignore DarkReader in a div

### DISCUSSION THREAD

**Discussion by rbrt-stn** [darkreader/darkreader#7318] [Q&A] [answered]
Hello, 

I am using the darkreader dependency for my personal React Web Application. I have a certain div where I don't want to apply the darkreader style. Could you please give me a suggestion how to achieve this?

According to https://github.com/darkreader/darkreader/issues/4056, I tried to add the classname darkreader, but without success. 
JSX Code:
`<div className='header darkreader'>`

CSS Code:
`.header.darkreader{
color: black;
}`

**Comment by Gusted** [comment, marked as answer]
Hi @rbrt-stn!

It's not possible to ignore specific `div`.

Also according to #4056, you can only apply the `darkreader` class to `<style>` which then would be ignored.

**Reply by rbrt-stn** [reply]
Thank you for the quick reply! 
In React I don't think it's possible to 
> apply the darkreader class to <style>


In the end I configured the enable function, as presented here [#4056 (comment)](https://github.com/darkreader/darkreader/issues/4056#issuecomment-723554928) (the second option).
Here is a template, in case somebody else needs it:
```
enable(
            {
                brightness: <number_value>,
                contrast: <number_value>,
                sepia: <number_value>
            },
            {
                ignoreInlineStyle: [],
                css: '.<class_name> { <css_attribute>: <value_for_css_attribute> }',
                ignoreImageAnalysis: [],
                invert: []
            }
        );
```
Thank you for providing such an amazing library! 🙏

**Reply by ShahriarKh** [reply]
Where can I find more info about `enable` API? I can't understand what should be put inside those arrays.

**Comment by Gusted** [comment]
> Where can I find more info about `enable` API? I can't understand what should be put inside those arrays.

https://github.com/darkreader/darkreader/blob/master/index.d.ts

**Reply by ShahriarKh** [reply]
Thanks! I believe these options should be noted in the README too.
